pkg/controllers/vdb/dbremovenode_reconciler.go

```diff
@@ -135,13 +136,20 @@ func (d *DBRemoveNodeReconciler) removeNodesInSubcluster(ctx context.Context, sc
 	return ctrl.Result{Requeue: requeueNeeded}, nil
 }
 
-// execATCmd will run the admintools command to remove the node
+// execAdminCmd will run the admin command to remove the node
 // This handles recording of the events.
-func (d *DBRemoveNodeReconciler) execATCmd(ctx context.Context, atPod types.NamespacedName, podNames string, cmd []string) error {
+func (d *DBRemoveNodeReconciler) execAdminCmd(ctx context.Context, atPod *PodFact, pods []*PodFact) error {
```

Comment:
Why not just `runRemoveNode` or `removeNode`?

Reply:
I'll use runRemoveNode